WSEvents.cpp

```diff
@@ -642,6 +653,55 @@ void WSEvents::StreamStatus() {
     obs_output_release(stream_output);
 }
 
+void WSEvents::Heartbeat() {
```

Comment:
Please write the event's specification as a comment above. These are used to generate the protocol specification document.
You can take one of the specification comment blocks of one of the other events as an example